### PR TITLE
Added --no-global-tag flag

### LINKED ISSUE #47: Is there any way to disable global version tag?
Is there any way to disable the global version tag?

When having multiple crates in the workspace that have a different release schedule I want to be able to disable the global version tag on minor/patch releases and turn it back on when making a major release. 

Is it currently possible?

### PR DESCRIPTION
**Pull request by vitalka200**
Added --no-global-tag flag to disable a global tag creation for
workspace. Resolves #47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,6 +173,7 @@ OPTIONS:
     -m, --message <message>                 Use a custom commit message when creating the version commit
         --pre-id <identifier>               Specify prerelease identifier
         --tag-prefix <prefix>               Customize tag prefix (can be empty) [default: v]
+        --no-global-tag                     Do not create a global tag for a workspace
```
Comment (pksunkara) [inline README.md:176]: This is actually in `FLAGS`. Also please do the same in `Publish` section of the readme too

**Review by pksunkara — CHANGES_REQUESTED** [review]
Looks good but minor readme changes needed

File: README.md (hunk 1 above)
Comment (vitalka200) [reply README.md:176]: right. I'll change it

Comment (vitalka200) [reply README.md]: Done

**Review by pksunkara — APPROVED** [review]

**Pull request merged**